core/ledger/pvtdatastorage/store.go

```diff
@@ -476,6 +532,90 @@ func (s *Store) GetPvtDataByBlockNum(blockNum uint64, filter ledger.PvtNsCollFil
 	return blockPvtdata, nil
 }
 
+func (s *Store) retrieveLatestPurgeKeyCollMarkerHt(ns, coll string) (*version.Height, error) {
+	encVal, err := s.db.Get(
+		encodePurgeMarkerCollKey(
+			&purgeMarkerCollKey{
+				ns:   ns,
+				coll: coll,
+			},
+		),
+	)
+	if err != nil {
+		return nil, err
+	}
+	if encVal == nil {
+		return nil, nil
+	}
+	return decodePurgeMarkerVal(encVal)
+}
+
+func (s *Store) keyPotentiallyPurged(k *dataKey) (bool, error) {
```

Comment:
We are never going to remember what 'potentiallyPurged' means, could you add a comment explaining it?

Reply:
Done